hbase-client/src/main/java/org/apache/hadoop/hbase/client/HBaseHbck.java

```diff
@@ -173,6 +174,7 @@ private static String toCommaDelimitedString(List<String> list) {
   }
 
   @Override
+  @Deprecated
   public Map<String, MasterProtos.RegionErrorType>
```

Comment:
How do we deal with this one?

Reply:
My plan is only keep "fixing" method in Hbck interface. And add a chore thread to check wrong stuff in master and show them in master web ui. And now we can use HBCK1 to check these.

Reply:
OK, got it. I saw your new PR #373